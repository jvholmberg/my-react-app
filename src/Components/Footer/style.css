.Footer-Container {
  background-color: #343a40;
  text-align: center;
}
.Footer-Links {
  display: flex;
  margin: 0 auto;
  padding: 3rem 3rem 0;
  width: 100%;
  max-width: 720px;
  justify-content: space-between;
  box-sizing: border-box;
}
.Footer-Links a {
  font-size: 1.25rem;
  color: #fff;
  text-decoration: none;
  text-transform: uppercase;
}
.Footer-Container .Footer-Links a:hover {
  color: #91979c;
}

.Footer-Information {
  margin: 0 auto;
  padding: 3rem 3rem 0;
  width: 100%;
  max-width: 720px;
  box-sizing: border-box;
  color: #fff;
  font-size: 1rem;
  font-weight: 300;
}
.Footer-Container .Footer-Social {
  padding: 3rem 1rem;
}
.Footer-Container .Footer-Social a {
  margin: 0 .5rem;
  font-size: 2rem;
  color: #fff;
}
.Footer-Container .Footer-Social a:hover {
  color: #91979c;
}

.Footer-Copyright {
  padding: 1rem;
  background: #25292d;
  color: #fff;
}



/* Medium devices (tablets, 768px and up) */
@media (max-width: 720px) {
  .Footer-Links {
    display: block;
  }
  .Footer-Container .Footer-Links a {
    display: block;
    margin: 1rem auto;
  }
}