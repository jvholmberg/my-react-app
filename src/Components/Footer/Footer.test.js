import React from 'react';
import renderer from 'react-test-renderer';

import Footer from './';

describe('Footer', () => {
  it('Empty matches snapshot', () => {
    const tree = renderer
      .create(<Footer />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
