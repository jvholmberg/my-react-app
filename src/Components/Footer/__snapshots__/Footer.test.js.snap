// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Empty matches snapshot 1`] = `
<footer
  className="Footer-Container"
>
  <div>
    <div
      className="Footer-Links"
    >
      <a
        href="dashboard"
      >
        About us
      </a>
      <a
        href="dashboard"
      >
        Products
      </a>
      <a
        href="dashboard"
      >
        Awards
      </a>
      <a
        href="dashboard"
      >
        Help
      </a>
      <a
        href="dashboard"
      >
        Contact
      </a>
    </div>
    <div
      className="Footer-Information"
    >
      <p>
        Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur aut odit aut fugit, sed quia consequuntur.
      </p>
    </div>
    <div
      className="Footer-Social"
    >
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-facebook-f"
        />
      </a>
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-twitter"
        />
      </a>
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-google-plus-g"
        />
      </a>
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-linkedin-in"
        />
      </a>
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-instagram"
        />
      </a>
      <a
        href="www.google.com"
      >
        <i
          className="fab fa-pinterest"
        />
      </a>
    </div>
    <div
      className="Footer-Copyright"
    >
      <span>
        Â© 2018 Copyright
      </span>
    </div>
  </div>
</footer>
`;
